packages/svelte/src/internal/client/proxy.js

```diff
@@ -84,7 +84,7 @@ function unwrap(value, already_unwrapped = new Map()) {
 		} else {
 			/** @type {Record<string | symbol, any>} */
 			const obj = {};
-			const keys = object_keys(value);
+			const keys = Reflect.ownKeys(value);
```

Comment:
our symbols need to be removed from those keys, I suspect the tests are failing now due to the same reason that my related PR around stale proxy references fixes